### PR TITLE
sql: add trace statement document

### PR DESCRIPTION
**Pull request by tiancaiamao**
Update document
Related PR: https://github.com/pingcap/docs-cn/pull/1110

### CODE REVIEW

File: sql/util.md [hunk 1]
```diff
@@ -97,3 +97,45 @@ USE db_name
...
+|   ├─session.CommitTxn     | 19:54:35.311601 | 14.696µs   |
+|   ├─recordSet.Next        | 19:54:35.311828 | 419.797µs  |
+|   ├─tableReader.Next      | 19:54:35.311834 | 379.932µs  |
+|   ├─recordSet.Next        | 19:54:35.312310 | 26.831µs   |
+|   └─tableReader.Next      | 19:54:35.312314 | 2.84µs     |
++---------------------------|-----------------|------------+
+10 rows in set (0.00 sec)
+```
+
+When the `format` is `json`, the output is some `json` text. If the text is too long, they are split into multiple lines.
```
Comment (lilin90) [inline sql/util.md]:
```suggestion
When the `format` is `json`, the output is some `json` text. If the text is too long, it is split into multiple lines.
```

File: sql/util.md [hunk 2]
```diff
@@ -97,3 +97,45 @@ USE db_name
...
+
+The `json` output can be viewed in the Web UI which is integrated in TiDB, here is an demo:
```
Comment (lilin90) [inline sql/util.md]:
```suggestion
The `json` output can be viewed in the Web UI which is integrated into TiDB. Here is a demo:
```

**Comment by ngaut** [comment]
PTAL @morgo

**Comment by lilin90** [comment]
@YiniXu9506 Could we use .gif in documentation if we want to generate it to PDF later?

**Comment by YiniXu9506** [comment]
Sure. I am doing research. I'll let you know once I find the way to solve this.

**Comment by lilin90** [comment]
@YiniXu9506 Any update? If it is inappropriate, we can use an ordinary image.

**Comment by YiniXu9506** [comment]
> @YiniXu9506 Any update? If it is inappropriate, we can use an ordinary image.

**Comment by YiniXu9506** [comment]
@lilin90 I've tried some ways to convert .gif to .png, but they all failed at generating PDF. I would suggest to use an ordinary image now. And I will keep doing research on different PDF generator to find a better way to convert .gif to .png.

**Comment by lilin90** [comment]
@YiniXu9506 Thanks a lot! @tiancaiamao So could you please replace the .gif file with ordinary images?

File: sql/util.md [hunk 3]
```diff
@@ -97,3 +97,45 @@ USE db_name
...
+|   ├─tableReader.Next      | 19:54:35.311834 | 379.932µs  |
+|   ├─recordSet.Next        | 19:54:35.312310 | 26.831µs   |
+|   └─tableReader.Next      | 19:54:35.312314 | 2.84µs     |
++---------------------------|-----------------|------------+
+10 rows in set (0.00 sec)
+```
+
+When the `format` is `json`, the output is some `json` text. If the text is too long, it is split into multiple lines.
+
+The `json` output can be viewed in the Web UI which is integrated into TiDB. Here is a demo:
```
Comment (morgo) [inline sql/util.md]: We should explain what the web UI is with consistent terminology.  For example:
```suggestion
A visualization of the `json` output can be accessed via the `tidb-server` status and metrics port (default: `:10080`). Here is a demo:
```

**Comment by morgo** [comment]
Sorry for the delay reviewing.  I made a quick comment, otherwise LGTM

File: sql/util.md (hunk 3 above)
Comment (tiancaiamao) [reply sql/util.md]: Done

**Comment by tiancaiamao** [comment]
Now it looks like this https://github.com/pingcap/docs/blob/e98d6cb524b3c442ce2a6b91badccf14d7697d23/sql/util.md

PTAL @morgo @lilin90

**Comment by morgo** [comment]
LGTM

**Pull request merged**